README.md

```diff
@@ -21,38 +21,54 @@ Here is a little demo
 
 * Holds various resource files that may be used by SlicerCineTrack like Icon and UI elements
 
-## Quick Setup Instructions for developer
```

Comment:
I think we should keep setup for developers and add a section for set up for users (before developers section). For normal user, they will just download the extension from catalog + sample dataset from our repo/releases as you add below.